### ISSUE TITLE
Dark/Night Mode

### ISSUE THREAD

**Issue by AntTargett** [disposedtrolley/literary-screensaver#1]
Hey James, 
Awesome project, a feature I would love to see is a dark/night mode
Any chance of implementing it?

**Comment by disposedtrolley** [comment]
Looks like it's possible to include a "configuration sheet" for the screensaver which defines the different options that can be selected - https://developer.apple.com/documentation/screensaver/screensaverview/1512486-configuresheet

Once that's worked out, it would be a case of choosing the colours for the dark mode :)

**Comment by disposedtrolley** [comment]
0.3-beta has been released with a default dark mode experience - https://github.com/disposedtrolley/literary-screensaver/releases/tag/0.3-beta

**Comment by keithamoss** [comment]
I'm happy to make a start on the configSheet UI if you haven't already.

Disclaimer: I've never made a screensaver, written Swift, or used XCode before. So I have no idea what I'm doing :)

**Comment by keithamoss** [comment]
Yeah, compared to what I'm used to in Python/Web/et cetera land I'm a little shocked to find how woeful the official Apple doco is.

There's a few nice reference projects like https://github.com/erikdoe/swift-circle that which look like they'll provide a sound basis to crib from.

**Comment by disposedtrolley** [comment]
It’s pretty great for iOS/UIKit stuff. I think there’s been a lack of focus on macOS dev in the past few years sadly. Good luck and shout out if you need any help!
